index.js

```diff
@@ -1,6 +1,7 @@
-const googleapis = require('googleapis')
+const Perspective = require('perspective-api-client')
 
 module.exports = robot => {
+  robot.perspective = new Perspective(({apiKey: process.env.PERSPECTIVE_API_KEY}))
```

Comment:
Assigning the `Perspective` object on robot makes it easy to mock the client in tests.